ci/jenkins/test.sh

```diff
@@ -74,7 +75,8 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --proxyall               Enable proxyAll to test AntreaProxy.
         --build-tag              Custom build tag for images.
         --docker-user            Username for Docker account.
-        --docker-password        Password for Docker account."
+        --docker-password        Password for Docker account.
+        --traffic-mode           Traffic Encryption mode."
```

Comment:
Seems like traffic mode net new in this PR, is that right? Perhaps we can specify the only support parameter (unless there are others?)

```suggestion
        --traffic-mode           Traffic Encryption mode: Only `wireGuard` is supported"
```

Reply:
@petertran-avgo , ipsec encryption is also supported however it needs additional kernel module on k8 node as prerequisites,  it could be preferred to keep description generic to provide future e2e tests for ipsec.